### PR TITLE
Jv 72 splash screen

### LINKED ISSUE #72: Create Splash screen
**Is your feature request related to a problem? Please describe.**
There is no way of seeing who worked on the game

**Describe the solution you'd like**
Have a splash screen that pops up on app load and does two things.
1. Hides the app assets and fonts loading in the background
2. Shows Me, Jorge, Kenny and Kendrick worked on the code and gave the ideas

### PR DESCRIPTION
**Pull request by JorgeLVilla**
## Changes
_List Changes Introduced by this PR_
1. Creates a new splash screen component to include the credits to the devs.
2. Adds a "Credits" button to main page.

## Purpose
User can now see who worked on the app.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #72

### CODE REVIEW

File: components/SplashScreen.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  StyleSheet,
+  Pressable,
+  View,
+} from "react-native";
+import { Link } from "react-router-native";
+import luis8bit from "../assets/luis8bit.png";
+import kenny8bit from "../assets/kenny8bit.png";
+import jorge8bit from "../assets/jorge8bit.png";
+import kendrick8bit from "../assets/kendrick8bit.png";
+
```
Comment (dannysanchez559) [inline components/SplashScreen.js:15]: :joy:

File: components/SplashScreen.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+const SplashScreen = () => {
+  return (
+    <SafeAreaView style={styles.background}>
+      {/* <Image source={require("../assets/cloudBackground.png")} /> */}
```
Comment (Lap343) [inline components/SplashScreen.js:19]: Youre going to want to remove this

**Review by dannysanchez559 — APPROVED** [review]
looks good man, cant wait to see.

**Review by Lap343 — APPROVED** [review]
Nice job just one line needs to be removed

File: components/SplashScreen.js (hunk 2 above)
Comment (JorgeLVilla) [reply components/SplashScreen.js:19]: Thanks Luis, good catch.

**Review by Gjhnsn — APPROVED** [review]

**Pull request merged**